control/config.py

```diff
@@ -61,6 +61,22 @@ def use_fbs_defaults():
 
 # Decide whether to use numpy.matrix for state space operations
 def use_numpy_matrix(flag=True, warn=True):
+    """Turn on/off use of Numpy `matrix` class for state space operations.
+
+    Parameters
+    ----------
+    flag : bool
+        If flag is `True` (default), use the Numpy (soon to be deprecated)
+        `matrix` class to represent matrices in the `~control.StateSpace`
+        class and functions.  If flat is `False`, then matrices are represnted
```

Comment:
typo in represented

Reply:
Thanks.  Will fix that in a subsequent PR.